test.py

```
@@ -0,0 +1,132 @@
+    )
+
+    simple_llm_agent = LLM(
+        llm_provider,
+        "You are the Code Review Agent, review the code given in the query. Answer the following query: {query}",
+    )
+
+    adaptive_agent = AdaptiveAgent(
+        llm_provider, prompt_provider, blast_radius_agent, AgentType.CODE_CHANGES
+    )
+
+    auto_router_agent = AutoRouterAgent(
```

Ruff (0.8.2)

Local variable `auto_router_agent` is assigned to but never used

Remove assignment to unused variable `auto_router_agent`

(F841)